questions.txt

```diff
@@ -0,0 +1,10 @@
+{ burn down 1/2 of the forest ; yes : it gets burnt down; no : it doesnt get burnt down}
+{ hire more staff ; hire 1 : you hire a worker; hire 2 : you hire 2 workers; hire 3 : you hire three workers}
+{ hire protestors ; no : nothing changes; yes : small increase of awareness; alot : large increase of awareness}
+{ licesnce removal of forest ; yes : gain  $1,000-$10,000; no : public approval increases ; alot : gain $10,000-$100,000}
+{ invest in other companies? ; no : nothing ; a little : small immedieate decrease in $ later posibility of some$ ; more : middle ; most : large immedieate decrease in $ later posibility of larger $}
+{ a hospital needs space for ading the sick; yes : public approval increases ; no : public approval idecreases}
+{ your forest is burning, give money for forest fighting; none : it burns ; $10,000 : decreases somewhaT ; $100,000 : decreases significantly}

```

Comment:
I really like this one! Makes you choose between two resources which is a good dynamic for the player